apis/v1alpha1/opentelemetrycollector_types.go

```diff
@@ -248,6 +248,16 @@ type ScaleSubresourceStatus struct {
 	// OpenTelemetryCollector's deployment or statefulSet.
 	// +optional
 	Replicas int32 `json:"replicas,omitempty"`
+
+	// readyReplicas is the number of pods targeted by this OpenTelemetryCollector with a Ready Condition.
+	// +optional
+	ReadyReplicas int32 `json:"readyReplicas,omitempty"`
+
+	// statusReplicas is the number of pods targeted by this OpenTelemetryCollector
+	// with a Ready Condition and non-terminated pdos targeted by this OpenTelemetryCollector's
```

Comment:
I am not sure if I get the description. Is it ready pods plus non-terminated pods? 

s/pdos/pods